project/iterative_sorting.py

```diff
@@ -1,36 +1,54 @@
 # Complete the selection_sort() function below in class with your instructor
+sort_array = [5, 7, 8, 4, 10, 7, 11, 18, 2019, 48]
+sort_array2 = [50, 70, 80, 40, 100, 70, 110, 180, 20190, 480]
+
 def selection_sort( arr ):
     # loop through n-1 elements
     for i in range(0, len(arr) - 1):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
-             
-
-
-
+        for j in range(cur_index, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
         # TO-DO: swap
-
-
-
+        temp = arr[smallest_index]
+        arr[smallest_index] = arr[cur_index]
+        arr[cur_index] = temp
```

Comment:
You can do this swap in a single line, like so: `arr[smallest_index], arr[cur_index] = arr[cur_index], arr[smallest_index]` in Python.

Reply:
I took your suggestion. Way simpler for sure.